planetiler-core/src/main/java/com/onthegomap/planetiler/worker/Worker.java

```diff
@@ -41,6 +41,20 @@ public Worker(String prefix, Stats stats, int threads, RunnableThatThrows task)
     this(prefix, stats, threads, i -> task.run());
   }
 
+  /**
+   * Constructs a new worker and immediately starts {@code threads} thread all running {@code task}.
+   *
+   * @param prefix       string ID to add to logs and stats
+   * @param stats        stats collector for this thread pool
+   * @param threads      number of parallel threads to run {@code task} in
+   * @param executorType the type of executor to use
+   * @param task         the work to do in each thread
+   */
+  @SuppressWarnings("java:S1181")
+  public Worker(String prefix, Stats stats, int threads, ExecutorType executorType, RunnableThatThrows task) {
+    this(prefix, stats, threads, executorType, i -> task.run());
+  }
```

Comment:
Let's remove this as well since we're just doing the fixed threads. I think things using virtual threads going forward will probably use a slightly different paradigm (new virtual thread per task, with a semaphore to limit concurrency) so probably don't need to be supported in the worker framework.